app/views/courses/index.html.erb

```diff
@@ -32,37 +32,40 @@
     <% end %>
 </div>
 <script type="text/javascript">
-  $('.attendance-present').on('click', function(event) => {
+  $('mark.item-key').on('click', alert('here'));
+
+  $('.attendance-present').on('click', function(event){
     $(event.target).addClass('rotate')
-  })
-  $('.attendance-icon').on('click', function(event) => {
+  });
+
+  $('.attendance-icon').on('click', function(event) {
     $(event.target).addClass('visible')
-  })
+  });
 
-  $(.attendance-icon).qtip({
+  $('.attendance-icon').qtip({
     content: {
       text: 'This is popping up'
-      <div class="attendance-icon attendance-present"></div><h4>Present</h4>
-      <div class="attendance-icon attendance-absent"></div><h4>Absent</h4>
-      <div class="attendance-icon attendance-tardy"></div><h4>Tardy</h4>
-      <div class="attendance-icon attendance-present_no_response"></div><h4>Present No Response</h4>
-      <div class="attendance-icon attendance-absent_with_response"></div><h4>Absent With Response</h4>
+      // <div class="attendance-icon attendance-present"></div><h4>Present</h4>
+      // <div class="attendance-icon attendance-absent"></div><h4>Absent</h4>
+      // <div class="attendance-icon attendance-tardy"></div><h4>Tardy</h4>
+      // <div class="attendance-icon attendance-present_no_response"></div><h4>Present No Response</h4>
+      // <div class="attendance-icon attendance-absent_with_response"></div><h4>Absent With Response</h4>
     },
     position: {
       target: 'mouse',
       adjust: { mouse: false }
     }
   })
 
-  <!-- $('.attendance-icon').qtip({
-    content: {
-        text: 'Loading...', // The text to use whilst the AJAX request is loading
-        ajax: {
-            url: '/path/to/file', // URL to the local file
-            type: 'POST', // POST or GET
-            data: {} // Data to pass along with your request
-        }
-    } -->
-});
+//   <!-- $('.attendance-icon').qtip({
```

Comment:
Sorry this stuff got pushed. I thought I deleted it at one point, but not the pushed version.